### DISCUSSION TITLE
Not getting out of the box DGS metrics

### DISCUSSION THREAD

**Discussion by AyushChaubey** [Netflix/dgs-framework#1381] [Q&A]
Hello,

After we upgraded the DGS version to `5.4.4`, we have stopped getting the DGS metrics that we get out of the box from `graphql-dgs-spring-boot-micrometer` module. If we fallback to the previous version `5.4.3`, we get the metrics correctly.

Has anything changed in the way metrics are exposed or any new property that we need to add in the config ?

**Comment by srinivasankavitha** [comment]
This should work just fine. Please ensure you have added the following dependencies to have this working:
```
    // ----
    // Enabling Metrics!
    //
    // First, we need to enable Spring Boot's actuators.
    implementation("org.springframework.boot:spring-boot-starter-actuator")
    // Second, enable DGS micrometer metrics integration.
    implementation("com.netflix.graphql.dgs:graphql-dgs-spring-boot-micrometer")
```

**Reply by AyushChaubey** [reply]
Thanks for your reply. The issue was related to [this](https://github.com/Netflix/dgs-framework/issues/1281#issuecomment-1287406079). As recommended, we have forced the version of graphql-java to 19.2. We are getting the metrics again.
